frontend/src/static/js/v_authorship.js

```diff
@@ -193,7 +181,7 @@ window.vAuthorship = {
 
     getFileBlankLineInfo(fileType) {
       return fileType + ': ' + 'Blank: ' + this.filesBlankLinesObj[fileType]
-          + ', Non-Blank: ' + (this.filesLinesObj[fileType] - this.filesBlankLinesObj[fileType]);
+          + ', Non-Blank: ' + (this.info.filesLinesObj[fileType] - this.filesBlankLinesObj[fileType]);
```

Comment:
why is it that `filesLinesObj` and `filesBlankLinesObj` are now stored in different level of abstraction?

Reply:
```filesLinesObj``` is now processed in the backend, therefore it is part of ```info``` prop now. However, all blank lines related information is still processed in the frontend, therefore ```filesBlankLinesObj``` is not part of the ```info``` prop.